### PR TITLE
GORA-488 Removed unused code. Edited the releated test codes.

### PR DESCRIPTION
**Pull request by cguzel**

### CODE REVIEW

File: gora-core/src/test/java/org/apache/gora/GoraTestDriver.java [hunk 1]
```diff
@@ -93,26 +88,47 @@ public void tearDown() throws Exception {
...
 
-  protected void setProperties(Properties properties) {
+  /**
+   * Instantiate a new {@link org.apache.gora.store.DataStore}. Uses default properties.
+   *
+   * @param keyClass        The key class.
+   * @param persistentClass The value class.
+   * @param <K>             class to be used for the key
+   * @param <T>             class to be persisted within the store
+   * @return
```
Comment (lewismc) [inline gora-core/src/test/java/org/apache/gora/GoraTestDriver.java]: Can you document this return.

File: gora-core/src/test/java/org/apache/gora/GoraTestDriver.java [hunk 2]
```diff
@@ -93,26 +88,47 @@ public void tearDown() throws Exception {
...
+   * @throws GoraException
```
Comment (lewismc) [inline gora-core/src/test/java/org/apache/gora/GoraTestDriver.java]: Same here, please document.

File: gora-core/src/test/java/org/apache/gora/GoraTestDriver.java [hunk 3]
```diff
@@ -93,26 +88,47 @@ public void tearDown() throws Exception {
...
 
+  /**
+   *  Instantiate a new {@link org.apache.gora.store.DataStore}.
+   *
+   * @param keyClass The key class.
+   * @param persistentClass The value class.
+   * @param properties The properties to be used be the store.
+   * @param <K> class to be used for the key
+   * @param <T> class to be persisted within the store
+   * @return
```
Comment (lewismc) [inline gora-core/src/test/java/org/apache/gora/GoraTestDriver.java]: document

File: gora-core/src/test/java/org/apache/gora/GoraTestDriver.java [hunk 4]
```diff
@@ -93,26 +88,47 @@ public void tearDown() throws Exception {
...
+   * @throws GoraException
```
Comment (lewismc) [inline gora-core/src/test/java/org/apache/gora/GoraTestDriver.java]: document

**Comment by cguzel** [comment]
I have added the documents. @lewismc

**Pull request merged**